general/clients/installing-kodi.md

```diff
@@ -1,61 +1,101 @@
 ---
 uid: clients-kodi
-title: Kodi
+title: Installing Kodi Add-On
 ---
 
-# Kodi
+# Installing Kodi Add-On
+
+> [!NOTE]
+> It's highly recommended to install the `Kodi Sync Queue` plugin into the Jellyfin server as well.
+> This will keep your media libraries up to date without waiting for a periodic re-sync from Kodi.
+
 
 ## Installation Process
 
-1. Download the repository installer [here](https://repo.jellyfin.org/releases/client/kodi/repository.jellyfin.kodi.zip).
+### Install Add-On Repository
+
+The recommended install method is to use the official Jellyfin add-on repository.  This allows for easy install of the Jellyfin for Kodi add-on, as well as automatically keeping the add-on up to date with the latest version.  Any other Jellyfin related add-ons that may be built in the future will also be available in this repository.
+
+#### General Use Devices (PCs and Tablets)
+
+1. Download the repository installer found [here](https://repo.jellyfin.org/releases/client/kodi/repository.jellyfin.kodi.zip).
     * It will be saved as `repository.jellyfin.kodi.zip`
 2. Install the Jellyfin repository.
-    * Open Kodi and navigate to "Add-On Browser"
+    * Open Kodi, go to the settings menu,  and navigate to "Add-On Browser"
     * Select "Install from Zip File"
-        * If prompted, enter settings and enable "Unknown Sources", then go back
+        * If prompted, enter settings and enable "Unknown Sources", then go back to the Add-On Browser
     * Select the newly downloaded file and it will be installed
-3. Install Jellyfin for Kodi.
+
+#### "Embedded" Devices (Android TV, Firestick, and other TV Boxes)
+
+1. Open Kodi, go to the settings menu, and navigate to "File manager"
+    * Select "Add source"
+    * In the text box, enter `https://repo.jellyfin.org/releases/client/kodi/`
+    * Enter a name for the data source, such as "Jellyfin Repo" and select Ok
+2. From the settings menu, navigate to "Add-On Browser"
+    * Select "Install from Zip File"
+        * If prompted, enter settings and enable "Unknown Sources", then go back to the Add-On Browser
+    * Select the data source you just added
+    * Install `repository.jellyfin.kodi.zip`
+
+### Install Jellyfin for Kodi Add-On
+
+1. Install Jellyfin for Kodi.
     * From within Kodi, navigate back to "Add-On Browser"
     * Select "Install from Repository"
-    * Choose "Kodi Jellyfin Addons"
-    * "Video Add-ons"
+    * Choose "Kodi Jellyfin Addons", followed by "Video Add-ons"
```

Comment:
add-ons or addons? We have a mixture of both

Reply:
addons imo

Reply:
i think add-on would be correct, but i'm just aiming for consistency

https://www.merriam-webster.com/dictionary/add-on

Reply:
All of Kodi's menus say "add-ons", but our repository says "addons".  Which I don't want to change now because that'll be annoying for users

Reply:
I doubt they'd even notice either way lol

Reply:
Thinking more, the repository could be renamed, but it's not in git so would have to be manual.  It would just be changed for all future users, existing ones wouldn't change.

Need to unzip this file, change the label inside the xml file, and zip it back up. https://repo.jellyfin.org/releases/client/kodi/repository.jellyfin.kodi.zip.

I don't have shell access to those servers, so if somebody wants to do it then have at it